docs/permissions.md

```diff
@@ -418,3 +418,46 @@ Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3O
 ```http
 HTTP/1.1 204 No Content
 ```
+
+### POST /permissions/exists
```

Comment:
The list of IDs we want to fetch the sharing status for can be quite long, so we have to pass it in the body. GET request cannot have URLS...